hazelcast-jca/src/test/java/com/hazelcast/jca/AbstractDeploymentHzClientTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+* Copyright (c) 2008-2016, Hazelcast, Inc. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package com.hazelcast.jca;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.EmptyAsset;
+import org.jboss.shrinkwrap.api.spec.ResourceAdapterArchive;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+
+/**
+* base class for jca tests that rely on client connections to hazelcast
+*
+*/
+public class AbstractDeploymentHzClientTest {
```

Comment:
Yes, I will. Thanks for pointing it out

Thanks - Juan
> On 27 Jul, 2016, at 7:16 AM, Bilal Yasar <notifications@github.com> wrote:
> 
> In hazelcast-jca/src/test/java/com/hazelcast/jca/AbstractDeploymentHzClientTest.java <https://github.com/hazelcast/hazelcast-ra/pull/5#discussion_r72446985>:
> 
> > +
> > +import org.jboss.arquillian.container.test.api.Deployment;
> > +import org.jboss.shrinkwrap.api.ShrinkWrap;
> > +import org.jboss.shrinkwrap.api.asset.EmptyAsset;
> > +import org.jboss.shrinkwrap.api.spec.ResourceAdapterArchive;
> > +import org.jboss.shrinkwrap.api.spec.WebArchive;
> > +import org.junit.AfterClass;
> > +
> > +import com.hazelcast.core.Hazelcast;
> > +import com.hazelcast.core.HazelcastInstance;
> > +
> > +/**
> > +* base class for jca tests that rely on client connections to hazelcast
> > +*
> > +*/
> > +public class AbstractDeploymentHzClientTest {
> can you add test category please?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/hazelcast/hazelcast-ra/pull/5/files/ceb99f180f8644333c36903548d73b4572a85c38#r72446985>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AHQu1VUTzdNDDofZ9BWKs4H2u2zgUSQ-ks5qZ2g-gaJpZM4JBMua>.
>